src/open_inwoner/openklant/tests/test_signal.py

```diff
@@ -141,3 +145,55 @@ def test_update_user_after_login_skips_existing_email(self, m):
 
         self.assertTimelineLog("retrieved klant for user")
         self.assertTimelineLog("updated user from klant API with fields: phonenumber")
+
+    def test_create_klant_for_digid_user(self, m):
+        m.get(
+            f"{KLANTEN_ROOT}klanten?subjectNatuurlijkPersoon__inpBsn=123456789",
+            json={"count": 0, "results": []},
+        )
+        m.post(f"{KLANTEN_ROOT}klanten", json=self.klant_bsn)
+
+        user = UserFactory(
+            login_type=LoginTypeChoices.digid,
+            bsn="123456789",
+        )
+
+        request = RequestFactory().get("/dummy")
+        request.user = user
+        user_logged_in.send(User, user=user, request=request)
+
+        self.assertTimelineLog(
+            f"created klant ({self.klant_bsn['klantnummer']}) for user"
+        )
```

Comment:
If this log is present, it does imply that things worked, but I would also add assertions to check that `m.request_history` contains the expected requests (one for the GET and one for the POST, and no others)